CLAUDE.md

```diff
@@ -0,0 +1,269 @@
+# CLAUDE.md
+
+This file provides guidance to Claude Code (claude.ai/code) when working with code in this repository.
+
+## Development Commands
+
+### Building and Development
+- `npm run lint` - Run ESLint and TypeScript checks (max 0 warnings)
+- `npm run lint-fix` - Automatically fix linting issues
+- `npm run tsc:ci` - Run TypeScript checks for both web and native platforms
+- `npm run tsc:web` - TypeScript check for web platform only
+- `npm run tsc:native` - TypeScript check for native platform only
+- `npm run tsc:ci` - Run both web and native TypeScript checks
+- `npm run lint:ci` - Run ESLint without type checking
+- `npm run lint-fix` - Run ESLint with auto-fix
+- `npm run validate` - Validate npm dependencies
+- `make dev` - Start development server with webpack-dev-server
+- `make compile` - Build production bundles
+- `make clean` - Clean build directory
+- `make all` - Full build (compile + deploy)
+
+### Testing
+- `npm test` - Run full test suite using WebDriverIO
+- `npm run test-single -- <spec-file>` - Run single test file
+- `npm run test-dev` - Run tests against development environment
+- `npm run test-dev-single -- <spec-file>` - Run single test in dev mode
+
+
+### Language Tools
+- `npm run lang-sort` - Sort language files
+- `npm run lint:lang` - Validate JSON language files
+
+### Platform-Specific TypeScript
+- `npm run tsc-test:web` - Check for native files incorrectly included in web build
+- `npm run tsc-test:native` - Check for web files incorrectly included in native build
+
+## Architecture Overview
+
+### Multi-Platform Structure
```

Comment:
Add a mention about the `web` and `native` directories, since that's how we differentiate things. There is also `features/mobile` which is native only.